fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -804,13 +805,13 @@ public static FHIRSearchContext parseQueryParameters(Class<?> resourceType,
      * Common logic from handling a single queryParameterValueString based on its type
      */
     private static List<QueryParameterValue> processQueryParameterValueString(Class<?> resourceType, SearchParameter searchParameter, Modifier modifier,
-            String queryParameterValueString) throws FHIRSearchException, Exception {
+        String queryParameterValueString) throws FHIRSearchException, Exception {
```

Comment:
sorry, missed this one on first pass.  the extra indent here is needed to distinguish this as a continuation of the method signature and not the first line of the method
```suggestion
            String queryParameterValueString) throws FHIRSearchException, Exception {
```

Reply:
this should be fixed